src/main/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfig.java

```diff
@@ -41,140 +42,165 @@
 @JsonDeserialize(builder = SlackEventNotificationConfig.Builder.class)
 public abstract class SlackEventNotificationConfig implements EventNotificationConfig {
 
-	private final String regex = "https:\\/\\/hooks.slack.com\\/services\\/";
-	private final Pattern pattern = Pattern.compile(regex, Pattern.MULTILINE);
-	public static final String TYPE_NAME = "slack-notification-v1";
+    public static final String TYPE_NAME = "slack-notification-v1";
 
-	static final String FIELD_COLOR = "color";
-	static final String FIELD_WEBHOOK_URL = "webhook_url";
-	static final String FIELD_CHANNEL = "channel";
-	static final String FIELD_CUSTOM_MESSAGE = "custom_message";
-	static final String FIELD_USER_NAME = "user_name";
-	static final String FIELD_NOTIFY_CHANNEL = "notify_channel";
-	static final String FIELD_LINK_NAMES = "link_names";
-	static final String FIELD_ICON_URL = "icon_url";
-	static final String FIELD_ICON_EMOJI = "icon_emoji";
+    private final String regex = "https:\\/\\/hooks.slack.com\\/services\\/";
+    private final Pattern pattern = Pattern.compile(regex, Pattern.MULTILINE);
+    private static final long MAX_BACKLOG_MESSAGES = 49;
```

Comment:
Why are we using a default of `49` instead of like `50`? That seems odd.

Reply:
This number is subject to change, when we discover new facts . I will make it 50 for now. Yeah 49 is odd :)